test/features/wellknown_jwks.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Endpoint to retrieve all public keys of the IdP server. Returns the public key set in JWKS format
+
+  Request endpoint: GET /.well-known/jwks.json
+
+  Scenario: Successfully retrieves the IdP server's public keys
+    When The request to retrieve all public keys of the IdP server is sent
+    Then The operation returns a public key set in JWKS format
```

Comment:
Please add in the end empty line

Reply:
Sure, my bad. I will add it right after we discuss the second comment on Monday.